basic-features.md

```diff
@@ -1,80 +1,136 @@
+| **Transactions**                                                                                         | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [Async commit](/system-variables.md#tidb_enable_async_commit-new-in-v50)                                 | Y            | Y            | N            | N            | N            | N            |
+| [1PC](/system-variables.md#tidb_enable_1pc-new-in-v50)                                                   | Y            | Y            | N            | N            | N            | N            |
+| [Large transactions (10GB)](/transaction-overview.md#transaction-size-limit)                             | Y            | Y            | Y            | N            | N            | N            |
+| [Pessimistic transactions](/pessimistic-transaction.md)                                                  | Y            | Y            | Y            | Y            | Y            | Experimental |
+| [Optimistic transactions](/optimistic-transaction.md)                                                    | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Repeatable-read isolation (snapshot isolation)](/transaction-isolation-levels.md)                       | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Read-committed isolation](/transaction-isolation-levels.md)                                             | Y            | Y            | Y            | N            | N            | N            |
+| **Partitioning**                                                                                         | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [Range partitioning](/partitioned-table.md)                                                              | Y            | Y            | Y            | Y            | Y            | N            |
+| [Hash partitioning](/partitioned-table.md)                                                               | Y            | Y            | Y            | Y            | Y            | N            |
+| [List partitioning](/partitioned-table.md)                                                               | Experimental | Experimental | N            | N            | N            | N            |
+| [List COLUMNS partitioning](/partitioned-table.md)                                                       | Experimental | Experimental | N            | N            | N            | N            |
+| **Statistics**                                                                                           | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [CMSketch](/statistics.md)                                                                               | Deprecated   | Deprecated   | Y            | Y            | Y            | Y            |
```

Comment:
For CMSketch, it is pretty tricky. We deprecated CMSketch in version 2 stats, and version 2 stats is enabled by default from 5.1 only for those newly inited clusters, for the clusters which are upgraded from lower versions, version 1 stats is still used by default.

Reply:
Make sense, LGTM